src/wallet/rpcwallet.cpp

```diff
@@ -2738,6 +2738,8 @@ static RPCHelpMan createwallet()
         RPCExamples{
             HelpExampleCli("createwallet", "\"testwallet\"")
             + HelpExampleRpc("createwallet", "\"testwallet\"")
+            + HelpExampleCli("createwallet", "\"descriptors\" false false \"\" true true true")
+            + HelpExampleRpc("createwallet", "\"descriptors\", false, false, \"\", true, true, true")
```

Comment:
It would be nice to have a way to specify examples using `bitcoin-cli -named`. Passing a list of booleans like this is hard to understand and hard to get right "which one do i need to toggle".

Reply:
Totally! I'll look into it

Reply:
Done